content/java/concepts/strings/terms/substring/substring.md

```diff
@@ -0,0 +1,58 @@
+---
+Title: '.substring()'
+Description: 'Returns a part of the string specified through starting and an optional ending index. If ending index is not specified, then the substring expands to the end of original string.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Characters'
+  - 'Strings'
+  - 'Methods'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The **`.substring()`** method returns a part of the string specified through starting and an optional ending index. If the ending index is not provided, then the substring expands to the end of of original string.
+
+## Syntax
+
+```pseudo
+// Returns substring from startIndex to the end of string
+string.substring(int startIndex);
```

Comment:
This line should not be removed, because it gives an example of syntax with only starting index. This is also done for current `.substring()` term for JavaScript.